examples/resources/spectrocloud_cluster_vsphere/resource.tf

```diff
@@ -0,0 +1,115 @@
+data "spectrocloud_cluster_profile" "vmware_profile"{
+  name = "vmware-public-repo"
+  version = "2.0.0"
+  context = "tenant"
+}
+data "spectrocloud_cloudaccount_vsphere" "vmware_account"{
+  name = "gm-pcg-wop-d"
+}
+
+
+resource "spectrocloud_cluster_vsphere" "cluster" {
+  name               = "vsphere-picard-2"
+  cluster_profile_id = data.spectrocloud_cluster_profile.vmware_profile.id
+  cloud_account_id   = data.spectrocloud_cloudaccount_vsphere.vmware_account.id
+
+  cluster_rbac_binding {
```

Comment:
Let's remove rbac, namespaces and other things which are not related to the example.

Reply:
removed